scripts/rel/cut.sh

```diff
@@ -18,6 +18,8 @@ RELEASE_GIT_TAG is a 'v*' or 'e*' tag for example:
   v5.0.12
   e5.0.0-beta.6
 
+If \$LAST_TAG exists, this script will automatically generate the changelogs from \$LAST_TAG to the current HEAD
```

Comment:
Usually `LAST_TAG` can be found from `git describe --tags --match 'v*'` for CE
and `--match 'e*'` for EE
The 'v' or 'e' prefix should be the same as `$1`.
Would be nice to support a default value if `LAST_TAG` is not provided.